api/odk/helpers/aggregate-submissions.js

```diff
@@ -36,6 +62,13 @@ module.exports = function (formName, errorCallback, aggregateCallback) {
             });
             return;
         }
+
+        // if offset, we do pagination
+        if (typeof offset !== 'undefined' && offset !== null) {
```

Comment:
Same thing? This is a convention I adopted years ago from what the Coffeescript compiler would do.